GooglePay/src/test/java/com/braintreepayments/api/googlepay/GooglePayLauncherUnitTest.kt

```diff
@@ -0,0 +1,69 @@
+package com.braintreepayments.api.googlepay
+
+import androidx.activity.result.ActivityResultLauncher
+import androidx.activity.result.ActivityResultRegistry
+import androidx.activity.result.contract.ActivityResultContract
+import androidx.fragment.app.FragmentActivity
+import com.google.android.gms.wallet.PaymentDataRequest
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class GooglePayLauncherUnitTest {
+
+    private val activityResultLauncher: ActivityResultLauncher<GooglePayPaymentAuthRequestParams?> =
+        mockk<ActivityResultLauncher<GooglePayPaymentAuthRequestParams?>>(relaxed = true)
+    private val callback: GooglePayLauncherCallback? = mockk<GooglePayLauncherCallback>(relaxed = true)
+    private val activityResultRegistry: ActivityResultRegistry? =
+        mockk<ActivityResultRegistry>(relaxed = true)
+
+    @Before
+    fun beforeEach() {
+        every {
+            activityResultRegistry?.register(
+                any(),
+                any(),
+                any<ActivityResultContract<GooglePayPaymentAuthRequestParams, Any>>(),
+                any()
+            )
+        } returns activityResultLauncher
+    }
+
+    @Test
+    fun constructor_createsActivityLauncher() {
+        val expectedKey = "com.braintreepayments.api.GooglePay.RESULT"
+        val lifecycleOwner = FragmentActivity()
+
+        val registry = mockk<ActivityResultRegistry>(relaxed = true)
+        GooglePayLauncher(registry, lifecycleOwner, callback!!)
```

Comment:
The GooglePayLauncher constructor takes in all non null parameters:

```kotlin
class GooglePayLauncher internal constructor(
    registry: ActivityResultRegistry,
    lifecycleOwner: LifecycleOwner,
    callback: GooglePayLauncherCallback
)
```

So we should be able to make all of the test dependencies non null - without the `?`.  This way we won't have to use the `!!` to convert to a non null type. It's a little less important for a unit test, but you usually want to avoid the non null assertion if possible.

Here's a good read on null safety from the Kotlin docs: https://kotlinlang.org/docs/null-safety.html